src/routes/users.mjs

```diff
@@ -168,6 +154,66 @@ export default async function () {
     }
   })
 
+  // NEW GET ROLE TYPES FUNCTION
```

Comment:
same as above, user roles are established by design, they are not dynamic. You can probably delete this.